deploy/helm/templates/metrics-deployment.yaml

```diff
@@ -108,6 +108,9 @@ spec:
                   name: {{ template "common.secret" . }}
                   key: SOLARWINDS_API_TOKEN
                   optional: true
+          envFrom:
+            - configMapRef:
+                name: {{ include "common.fullname" (tuple . "-common-env") }}
```

Comment:
Use the environment variables for `CLUSTER_UID` and `OTEL_ENVOY_ADDRESS`, like we do in all other containers.